Header header
uint8 motor_num
#假设的7个关节的位置和速度
float64[7] position
float64[7] velocity
